zk_toolbox/crates/zk_supervisor/src/commands/test/revert.rs

```diff
@@ -48,7 +52,13 @@ fn run_test(
         cmd!(shell, "yarn mocha tests/revert-and-restart.test.ts")
     };
 
-    let mut cmd = Cmd::new(cmd).env("CHAIN_NAME", &ecosystem_config.default_chain);
+    let mut cmd = Cmd::new(cmd).env(
+        "CHAIN_NAME",
+        global_config()
+            .chain_name
+            .as_deref()
+            .unwrap_or(ecosystem_config.default_chain.as_ref()),
```

Comment:
should we add a method, `current_chain` to ecosystem config and implement this logic there?

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/ea4dd656aa26327948c257014365ca66b5894ef1